hr_holidays_legal_leave/README.rst

```diff
@@ -0,0 +1,78 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+HR Holidays Legal Leave
+=======================
+
+Allows the define what holiday type is to be considered legal/annual leave type.
```

Comment:
Allows to define which holiday type is to be considered legal/annual leave type.